rest/data/distro.go

```diff
@@ -117,7 +117,8 @@ func CreateDistro(ctx context.Context, u *user.DBUser, newDistroId string) error
 		Id:   newDistroId,
 		Arch: evergreen.ArchLinuxAmd64,
 		BootstrapSettings: distro.BootstrapSettings{
-			Method: distro.BootstrapMethodNone,
+			Method:        distro.BootstrapMethodLegacySSH,
+			Communication: distro.CommunicationMethodLegacySSH,
```

Comment:
I changed this because the code says that `BootstrapMethodNone` is for [internal use only](https://github.com/evergreen-ci/evergreen/blob/4055cd62aeebf77af84749e9eedd58a2547c71be/model/distro/distro.go#L279), and it's not a selectable option on the legacy UI page

Reply:
👍 Sounds good, and yeah, this one isn't an option that users or admins can directly use.